i18n/locales/en/common.json

```diff
@@ -517,6 +517,8 @@
   "[Tomorrow], hh:mm A": "[Tomorrow], hh:mm A",
   "[Yesterday], hh:mm A": "[Yesterday], hh:mm A",
   "from": "from",
+  "i.e. 10169746": "i.e. 10169746",
+  "i.e. shinekami": "i.e. shinekami",
```

Comment:
`10169746` and `shinekami` are static strings that don't need to be part of the translation strings. You can use `i.e {{value}}` instead of those top strings.